doc/rst/developer/bestPractices/buildingdocs.rst

```diff
@@ -1,15 +1,227 @@
 .. _readme-buildingdocs:
 
+.. _Chapel Developers: https://chapel.discourse.group/c/developers
```

Comment:
Minor, but I think of this as more like a footnote than something that should appear first in the file, so would either move it to the end of the file or the end of the section where it's referenced (for the sake of someone reading the file from its rst sources, which I think does happen sometimes).

Reply:
I think this was just a leftover copy and paste thing.  I deleted it from this location in the file.  It was already showing up later in the document.

Reply:
Unfortunately, I don't think that's the case.  The place it shows up later in the document is illustrating to a reader how to create a hyperlink (within a formatted code block), but it doesn't/can't actually create a hyperlink that can be clicked on from within the file, which this was doing.  For example, note that going to:

https://github.com/chapel-lang/chapel/blob/main/doc/rst/developer/bestPractices/buildingdocs.rst

we now have a broken link when clicking on the "Chapel Developers" link within the sentence:

> `Linking from one of the html files at https://chapel-lang.org such as https://chapel-lang.org/contributing.html can only be done by a core contributor. Post an inquiry to the `Chapel Developers`_ Discourse forum.`

(whereas with the version that I commented on here, it did lead to the Discourse forum).  So I think the link definition needs to be reintroduced, where I'd put it just at the end of that section so that it reads more like a footnote for someone browsing the .rst version of the file.

Reply:
Fixed in PR #18316